test/test_helpers/certificates.rb

```diff
@@ -28,31 +28,48 @@ def create_key(alg)
       end
     end
 
-    def create_cert(key = create_key(:rsa))
-      public_key = get_public_key(key)
+    def create_ca(key = create_key(:rsa))
```

Comment:
Hm... the description says that the working combination is DSA + SHA256, but I don't see anywhere where DSA is forced, it seems that everything defaults to RSA?...

Reply:
We have tests for each algorithm:

https://github.com/jlledom/apisonator/blob/b471437ee6a8915d1a00e51d8f3a6cc637b02ef0/test/unit/storage_async_test.rb#L234-L251

Before this PR, the tests for `:ec` and `:rsa` passed, but the test for `:dsa` failed with the `invalid digest` error.

Reply:
Ah, OK, that last phrase clarifies the issue. Thanks!

Reply:
Sorry for the confusion!